torch/sparse/_triton_ops.py

```diff
@@ -195,7 +196,7 @@ def _bsr_strided_dense_rowspace_kernel(
             + col_indices_stride * nnz_offset
         )
 
-        output_acc_block = tl.zeros((BLOCKSIZE_ROW, BLOCKSIZE_ROW), tl.float32)
+        output_acc_block = tl.zeros((BLOCKSIZE_ROW, BLOCKSIZE_ROW), dtype=acc_dtype)
         for _ in range(row_nnz):
             values_block = tl.load(values_block_ptrs)
 
```

Comment:
It won't let me add a suggestion outside of the diff, but playing with this locally the tests pass if you modify the `tl.dot` call found a few lines below:

```python
output_acc_block += tl.dot(values_block, dense_block, allow_tf32=False)
```

You can make toggling the flag conditional on the dtype in play, but as far as I know it should only have an effect for float32. 

Note that things do not quite work out if we set that flag to match the value of `torch.backends.cuda.matmul.allow_tf32`. When torch has this flag set true and the triton flag is also set true the results do not match, and the errors are too large for adjusting the dtype to account for them (max ~10^-1 absolute error).

Reply:
Thank you! I will play with it. I wonder whether it affects all mm-like methods...